tracer/src/Datadog.Trace/Ci/Test.cs

```diff
@@ -347,6 +349,17 @@ public void Close(TestStatus status, TimeSpan? duration, string? skipReason)
             case TestStatus.Skip:
                 tags.Status = TestTags.StatusSkip;
                 tags.SkipReason = skipReason;
+                if (tags.SkipReason == IntelligentTestRunnerTags.SkippedByReason)
+                {
+                    tags.SkippedByIntelligentTestRunner = "true";
+                    tags.TestsSkipped = "true";
+                    Suite.Tags.AddIntelligentTestRunnerSkippingCount(1);
+                }
+                else
```

Comment:
@gnufede , in https://github.com/DataDog/dd-trace-py/pull/6463/files#diff-68a18a0364b7a1b385c2dbcbf0025d16286889239f4557ea57d69eb663a2c914R567 , we only add the tag when the test's been skipped. Should we set it if false?